src/target/renesas.c

```diff
@@ -948,3 +992,100 @@ static bool renesas_uid(target_s *t, int argc, const char **argv)
 
 	return true;
 }
+
+static bool renesas_print_cmd_usage(target_s *t)
+{
+	tc_printf(t, "command help goes here\n");
+	return false;
+}
+
+static bool renesas_cmd_option(target_s *t, int argc, const char **argv)
+{
+	(void)argc;
+	(void)argv;
+
+	const uint8_t arglen = strlen(argv[1]);
+
+	if (arglen > 0 && strncmp(argv[1], "get", arglen) == 0) {
+		if (argc != 3)
+			return renesas_print_cmd_usage(t);
+
+		target_addr_t option_addr = strtoul(argv[2], NULL, 0);
+		if (!renesas_check_option_setting(t, option_addr))
+			return renesas_print_cmd_usage(t);
+
+		/* The address must be aligned */
+		if (option_addr % 0x10 > 0)
+			return renesas_print_cmd_usage(t);
+
+		uint32_t value = target_mem_read32(t, option_addr);
+
+		tc_printf(t, "Read from address: 0x%02x, value: %" PRIx32 "\n", option_addr, value);
+
+		return true;
+	} else if (arglen > 0 && strncmp(argv[1], "set", arglen) == 0) {
+		if (argc != 4)
+			return renesas_print_cmd_usage(t);
+
+		target_addr_t option_addr = strtoul(argv[2], NULL, 0);
+		if (!renesas_check_option_setting(t, option_addr))
+			return renesas_print_cmd_usage(t);
+
+		/* The address must be aligned */
+		if (option_addr % 0x10 > 0)
+			return renesas_print_cmd_usage(t);
+
+		/* Configuration-Set command always needs 16 bytes of data */
+		const uint8_t len = 16UL;
+		uint8_t data[len];
+
+		for (size_t i = 0; i < len;) {
+			uint32_t value = target_mem_read32(t, option_addr + i);
+			memcpy(&data[i], &value, 4UL);
+			i += 4UL;
+		}
+
+		uint32_t value = strtoul(argv[3], NULL, 0);
+		memcpy(&data, &value, sizeof(uint32_t));
+		tc_printf(t, "Write to address: 0x%02x, value: %" PRIx32 "\n", option_addr, value);
+
+		if (renesas_rv40_pe_mode(t, PE_MODE_CF)) {
+			/* Permit flash operations */
+			target_mem_write8(t, SYSC_FWEPROR, SYSC_FWEPROR_PERMIT);
+			/* set block start address */
+			target_mem_write32(t, RV40_FSADDR, option_addr);
+
+			/* issue two part Write commands */
+			target_mem_write8(t, RV40_CMD, RV40_CMD_CONFIG_SET_1);
+			target_mem_write8(t, RV40_CMD, RV40_CMD_CONFIG_SET_2);
+
+			platform_timeout_s timeout;
+			platform_timeout_set(&timeout, 200);
+
+			for (size_t i = 0U; i < len;) {
```

Comment:
Same comment as above about the loop increment value. Additionally, drop the braces from the loop and use `read_le2()` from the buffer_utils.h header for constructing the values to pass to `target_mem_write16()`.